src/views/RichWorkspace.vue

```diff
@@ -71,7 +71,7 @@ export default {
 			loaded: false,
 			ready: false,
 			autofocus: false,
-			darkTheme: OCA.Accessibility.theme === 'dark',
+			darkTheme: typeof OCA.Accessibility !== 'undefined' && OCA.Accessibility.theme === 'dark',
```

Comment:
Why not just  ?
```suggestion
			darkTheme: OCA && OCA.Accessibility && OCA.Accessibility.theme === 'dark',
```

Reply:
Well that still throws an ReferenceError

Reply:
OCA is not defined?

Reply:
Ah never mind, to early :see_no_evil: